config/GraduateAttributes.ts

```diff
@@ -290,6 +311,11 @@ export let resolveGraduateAttributes = (
         attribute,
         contributor.activityData?.authored_issue_and_pr?.length,
       );
+    case "empathy":
+      return resolveLevel(
+        attribute,
+        contributor.highlights.discussion_answered / 5,
```

Comment:
Why divide by 5?

Reply:
Highlights.discussion answered is point write ?
Now let the user have 50 points for discussion answered 
so he answered no of discissions is 50/5 10 okk?

Reply:
Divide by referencing that instead. What if the points is changed later on?

Reply:
![image](https://github.com/user-attachments/assets/2e6558de-52af-4aed-b150-d68eed644697)

Reply:
yup